build/Katana.version.targets

```diff
@@ -4,14 +4,14 @@
     <ShipVersion>4.0.0</ShipVersion>
     <ShipStrongNameVersion>4.0.0.0</ShipStrongNameVersion>
     <ShipFullVersion>4.0.0-rtm-000</ShipFullVersion>
-    <ShipFileVersion>4.0.70129.0</ShipFileVersion>
+    <ShipFileVersion>4.0.80104.0</ShipFileVersion>
     <MajorVersion>4</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
     <BuildQuality>rtm</BuildQuality>
-    <BuildDate>70129</BuildDate>
+    <BuildDate>80104</BuildDate>
     <BranchSuffix></BranchSuffix>
-    <Eula>https://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm</Eula>
+    <Eula>https://raw.githubusercontent.com/aspnet/AspNetKatana/v4.0.0/LICENSE.txt</Eula>
```

Comment:
Not so fast, mister! 😄 Can you update the main LICENSE.txt and this tagged one to be a copy of https://github.com/aspnet/AspNetCore/blob/master/LICENSE.txt?